client/my-sites/plan-features/footer.jsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import noop from 'lodash/noop';
+
+/**
+ * Internal dependencies
+ */
+import { localize } from 'i18n-calypso';
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+
+const PlanFeaturesFooter = ( { translate, current = false, description, onUpgradeClick = noop } ) => {
+	return (
+		<footer className="plan-features__footer">
+			<p className="plan-features__footer-desc">{ description }</p>
+			<div className="plan-features__footer-buttons">
+				{
+					current
+						? <Button className="plan-features__footer-button is-current" disabled>
```

Comment:
This matches our coding guidelines https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#multi-line-statements so the alternative might be to break this out to something like this:

```
{ current ? this.getCurrentPlanButton() : this.getUpdgradeButton() }
```